src/introduction/README.md

```diff
@@ -16,40 +18,40 @@ and software stacks.
 ## Skills development exercises
 
 **Skills development exercises** are intended to take 1-2 hours each,
-and ask you to build and perform minor modifications to simple
-RISC-V and CHERI-RISC-V C/C++ programs. These exercises
-facilitate building skills such as compiling, executing,
-and debugging RISC-V and CHERI-RISC-V programs, as well as to build basic
-understanding of CHERI C/C++ properties. We highlight some key edge
-cases in CHERI, including the effects of bounds imprecision, subobject
-bounds, weaker temporal safety, and C type confusion.
+and ask you to build and perform minor modifications to simple C/C++
+programs. These exercises facilitate building skills such as
+compiling, executing, and debugging programs with both CHERI enabled
+and default ISAs, as well as to build basic understanding of CHERI
+C/C++ properties. We highlight some key edge cases in CHERI, including
+the effects of bounds imprecision, subobject bounds, weaker temporal
+safety, and C type confusion.
 
 These exercises take for granted a strong existing understanding of:
 - The C/C++ languages
 - UNIX program compilation, execution, and debugging
-- RISC ISAs and binary structures/reverse engineering (e.g., on MIPS or ARMv8)
+- ARMv8 and/or RISC ISAs as well as binary structures/reverse engineering
```

Comment:
```suggestion
- RISC ISAs (e.g. Armv8-A or RISC-V) as well as binary structures/reverse engineering
```
Arm is a RISC ISA (or at least claims to be; more of a load/store architecture these days)